### ISSUE TITLE
UI strings in german language pack

### ISSUE THREAD

**Issue by johnnydvc** [Acce0ss/harbour-qr-share-plugin#3]
hey, nice app.
but I don't get how to install the separate language pack as mentioned in the description.
so far the pulley menu strings look a little bit raw
thanks

**Comment by eson57** [comment]
Yes, same here. Can't find where to put the .qm file. Usually there's an app folder at usr/share/, but none for qr-share-plugin.

**Comment by Acce0ss** [comment]
hello, simply download the rpm from openrepos.net, and install, no need to figure out the directories. For the record, the qm files are supposed to be located in /usr/share/translations/nemotransferengine/. Unfortunately the openrepos.net client, Warehouse, only shows one package, so it probably confused you two.

**Comment by Acce0ss** [comment]
Also, there is no german language translation as of yet, sorry

**Comment by eson57** [comment]
Thanks, working fine now. Except for the outdated Swedish language file. I uploaded adjusted file 9 days ago. 
https://svenskasprakfiler.se/tempfiles/harbour_qr_share_plugin_sv.ts
